### DISCUSSION TITLE
The method moment().isValid() in strict mode has some issues with MM/dd/YYYY format

### DISCUSSION THREAD

**Discussion by prox2** [moment/moment#6419] [Q&A] [answered]
Hello, 
The method moment().isValid() in strict mode has some issues with MM/dd/YYYY format as it will return false. However it will always return true when date of format MM/dd/YYYY has (Thursdays of December and September) or (Wednesdays of June) or (Saturday of January) or (Sundays of May) or (Mondays of August), for 2022 and change on 2023 and more example didn't discover (Not sure based on what it give validations true for specific dates in curtain months and curtain year where will be changed to another months in another year) , for example: 12/Th/2022, 09/Th/2022 will be always true in 2022 only where in 2023 will be false,  Other than that, all provided dates will return false validation value.

[Here](https://codesandbox.io/s/moment-js-playground-forked-t4527z): https://codesandbox.io/s/moment-js-playground-forked-t4527z I have tested moment with different cases.

**Comment by mattjohnsonpint** [comment, marked as answer]
Thanks for the examples! Moment tokens are case-sensitive: lowercase `dd` parses a two-letter weekday, while uppercase `DD` parses day of month. I'd use `moment(value, 'MM/DD/YYYY', true).isValid()`. That rejects values such as `12/Th/2022` consistently.
